pyproject.toml

```diff
@@ -46,7 +46,8 @@ dependencies = [
     "sympy>=1.10.1",
     "ubermagutil>=0.63.2",
     "vtk>=9.1",
-    "xarray"
+    "xarray",
+    "pyvista[all]>=0.42.3"
```

Comment:
Do we need `"all"` or would a more restrictive selection be sufficient?

Reply:
`all` also installs:
```python3
colormaps = [
    'cmocean',
    'colorcet',
]
io = [
    'imageio',
    'meshio>=5.2'
]
jupyter = [
    'ipywidgets',
    'jupyter-server-proxy',
    'nest_asyncio',
    'trame>=2.5.2',
    'trame-client>=2.12.7',
    'trame-server>=2.11.7',
    'trame-vtk>=2.5.8',
    'trame-vuetify>=2.3.1',
]
```

I think its easier than needing to do all the `trame` stuff ourselves. Let me know what you think.

Reply:
I agree, installing `all` is better/easier.